OrbitLinuxTracing/LibunwindstackUnwinder.cpp

```diff
@@ -1,20 +1,23 @@
 #include "LibunwindstackUnwinder.h"
 
-#include "PrintVar.h"
+#include <array>
+#include <cstdio>
+
+namespace LinuxTracing {
 
 bool LibunwindstackUnwinder::SetMaps(const std::string& maps_buffer) {
   maps_ = std::make_unique<unwindstack::BufferMaps>(maps_buffer.c_str());
 
   if (!maps_->Parse()) {
-    PRINT("LibunwindstackUnwinder::SetMaps: failed to parse maps\n");
+    fprintf(stderr, "LibunwindstackUnwinder::SetMaps: failed to parse maps\n");
```

Comment:
hide fprintfs behind ERROR/LOG() macros? So we can easly get __PRETTY_FUNCTION__ and __LINE__ int he future for these messages.

Parts of PrintVar.h can be used for that

Reply:
As discussed offline, created a Logging.h in the new module for now. It will be moved to a new OrbitBase in the future.